### ISSUE TITLE
It becomes unrespive when sftp cannot connect

### ISSUE THREAD

**Issue by jiangjianghu** [kingToolbox/WindTerm#874]
When I turn on file management and the ssh device doesn't have sftp, the terminal will die

**Comment by kingToolbox** [comment]
Duplicate of #477 

This is a bug that has existed for a long time, and it's time to solve it. I will start to fix this bug after the `2.5.0` version is released. Please stay tuned, thank you.

BTW:
Currently you can avoid this by unchecking the check box `Session Property Dialog - SSH - SSH - Authentication - Enable SFTP`.

**Comment by jiangjianghu** [comment]
ok,thanks, I wonder if it is possible to use scp instead of sftp, when ssh device doesn't have sftp

**Comment by kingToolbox** [comment]
No, `Scp` (Secure **Copy** Protocol) can only be used to **copy or transfer** files, only `SFtp` supports reading directories and files, deleting files, modifying file attributes, etc.
